addons/l10n_it_edi/tests/test_edi_import.py

```diff
@@ -129,3 +131,11 @@ def test_receive_same_vendor_bill_twice(self):
         self.assertEqual(len(attachments), 1)
         invoices = self.env['account.move'].search([('payment_reference', '=', 'TWICE_TEST')])
         self.assertEqual(len(invoices), 1)
+
+    def test_discount_line(self):
+        """Test discount line addition after bill import"""
+
+        content = etree.fromstring(self.invoice_discount_content)
+        invoices = self.edi_format._create_invoice_from_xml_tree(self.invoice_discount_file, content)
+
+        self.assertEqual(invoices.line_ids.name, 'SCONTO')
```

Comment:
If we do the entire test, we could check if the line values are correct entirely...